scapy/fields.py

```diff
@@ -1474,7 +1475,8 @@ class PacketListField(_PacketField[List[BasePacket]]):
     that might occur right in the middle of another Packet field.
     This field type may also be used to indicate that a series of Packet
     instances have a sibling semantic instead of a parent/child relationship
-    (i.e. a stack of layers).
+    (i.e. a stack of layers). All elements in PacketListField have current packet
+    referenced in parent field.
```

Comment:
```suggestion
    (i.e. a stack of layers). All elements in PacketListField have current
    packet referenced in parent field.
```
That should fix the flake8 failure